ui/components/app/flask/snap-install-warning/index.scss

```diff
@@ -5,6 +5,7 @@
     display: flex;
     align-items: flex-start;
     gap: 0 16px;
+    margin-bottom: 16px;
```

Comment:
Is this needed when having just one warning?

Reply:
Nope, changing that